cmd/generate/config/rules/openai.go

```diff
@@ -0,0 +1,24 @@
+package rules
+
+import (
+	"regexp"
+
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func OpenAI() *config.Rule {
+    // Define Rule
+    r := config.Rule{
+        Description: "Open AI token",
+        RuleID:      "open-ai-token",
+        Regex: regexp.MustCompile("(sk-)[a-zA-Z0-9]{20}(T3BlbkFJ)[a-zA-Z0-9]{20}"),
```

Comment:
`sk` isn't restrictive enough. Please use `generateSemiGenericRegex`

> `T3BlbkFJ`

What is this?

Reply:
I've repeatedly issued tokens on openAI and found that all tokens have a common "T3BlbkFJ" and that there exists a random string with a length of 20 before and after that.

Reply:
Hi @mkthebea, is it the same case if an account is using the GPT-4 model? Or is "T3BlbkFJ" common amongst all keys, iregardless of the model? Thanks!

Reply:
They are "OpenAI" tokens and not specific to any specific service, so the model shouldn't matter.